views/assets/js/App/DateDistance.js

```diff
@@ -0,0 +1,19 @@
+import $ from 'jquery';
+import distanceInWordsToNow from 'date-fns/distance_in_words_to_now';
+
+class DateDistance {
+    static calculate($elements) {
```

Comment:
I would name it `$elements` only if it is already a jQuery collection. And in such case, you don't need to wrap it again in jQuery.

Reply:
Dependency of jQuery for class DateDistance was removed.